package.json

```diff
@@ -12,6 +12,7 @@
     "autoprefixer": "^10.1.0",
     "css-loader": "^5.0.1",
     "css-minimizer-webpack-plugin": "^1.1.5",
+    "is-svg": "^4.3.1",
```

Comment:
Hmm... Do we need to add this dependency directly to the package.json or is this something we can update as an indirect dependency in the yarn.lock?

Reply:
That's a good question.  I wasn't able to get it to stick but modifying yarn.lock, but I haven't really worked with yarn before so I could have doing something wrong.  When I ran `yarn install` it overwrote the version with 3.0.0 in yarn.lock.  When I tried running yarn upgrade is-svg, it also stayed at 3.0.0. 🤷‍♀️

Reply:
Hmm looked into this and I think we need to wait for the dependency chain on `css-minimizer-webpack-plugin` to be updated before we can address the `is-svg` vulnerability, because it's a dependency of `cssnano` which is a dependency of `css-minimizer-webpack-plugin`. Let's remove this line from `package.json` for now since it doesn't really address the deeper security issue. I don't think it's a big deal at the moment.

See: https://github.com/webpack-contrib/css-minimizer-webpack-plugin/issues/60